tests/test_methods_and_attributes.py

```diff
@@ -98,6 +98,11 @@ def test_properties():
     instance.def_property = 3
     assert instance.def_property == 3
 
+    with pytest.raises(AttributeError):
+        dummy = instance.def_property_writeonly  # noqa: F841 unused var
```

Comment:
This should have a `assert "unreadable attribute" in str(excinfo)` after it (the later ones all have it, just this one looks to be missing the assertion value check).